DOCS/interface-changes/vapoursynth-api-v4.txt.txt

```diff
@@ -0,0 +1,3 @@
+Bump dependency of VapourSynth to utilize its API version 4. New minimum VapourSynth version for runtime is R56.
+
```

Comment:
it will be merged into one file later, I would keep it in one line. It will be wrapped automatically.